task-3-SPRINGBOOT-PETS/Tasks.md

```diff
@@ -0,0 +1,50 @@
+<h1> LeverX Sprint Boot Task </h1>
+
+<h2> Important information: </h2>
+
+1. You can find some useful materials in [Materials.md](/task-3-SPRINGBOOT-PETS/materials/Materials.md). 
+2. Pay attention to the file [Self-check.md](/task-3-SPRINGBOOT-PETS/Self-check.md).
+3. You are able to use *Lombok* and *other libraries* you think you need.
+4. It's convenient to use the following service to compose and share database schemas: https://dbdiagram.io/home
+5. Check Markdown in [Materials.md](/task-3-SPRINGBOOT-PETS/materials/Materials.md) for writing a nice documentation.
+6. Read about **HTTP response status codes**.
+
+<h2> Requirements: </h2>
+
+1. You have to implement task given below using **Java 8+**.
+2. You need to have installed [Maven](https://maven.apache.org/) for implementing this task.
+3. Use any **relational database**. 
+We can recommend using Postgres, but your choice is up to you.
+4. Use [Postman](https://www.postman.com/) to interact with your application and check API endpoints. 
+**Save your API collection** and stick it to the project
+(either as a link at README.md or as a file in directory 'postman' in the root of your project).
+5. **Stick database schema** to README.md as pictures or as a link.
```

Comment:
We can also link to any file in the root of this repo as an example/template of the README.md structure

Reply:
I think it is more logical to provide readme.md example in the servlet task, because that task is before this and interns should write readme file in servlet task as well as here